R/lib.R

```diff
@@ -21,6 +22,7 @@ create_loupe_from_seurat <- function(
   output_dir = NULL,
   output_name = NULL,
   dedup_clusters = FALSE,
+  feature_ids = NULL,
```

Comment:
should the README/docs be updated to reflect this new input + an explanation for what it's for + examples of how to use it?